Features/AdminCommands/MuteCommand.cs

```diff
@@ -0,0 +1,34 @@
+using ArtcordAdminBot.Features.Helpers;
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+
+namespace ArtcordAdminBot.Features.AdminCommands
+{
+    public partial class AdminCommandGroup
+    {
+        [Command("mute")]
+        [System.ComponentModel.Description("Mutes a user")]
+        public async Task MuteAsync(CommandContext ctx,
+        [System.ComponentModel.Description("The user to mute.")] DiscordUser targetUser)
+        {
+            var mutedRoleId = await _databaseService.GetMutedRoleAsync(ctx.Guild!.Id);
```

Comment:
Don't use `var` for primitive types (not technically a given standard, I just don't like it and think it makes the code less readable)

```suggestion
            ulong mutedRoleId = await _databaseService.GetMutedRoleAsync(ctx.Guild!.Id);
```